Remove TS compatibility transition code

mikekgfb opened the pull request:
Summary: Remove TS compatibility transition code

Test Plan: sandcastle

Differential Revision: D39494677

erichan1 reviewed (approved):
Hooray! Remove code == good!

mikekgfb commented:
@pytorchbot merge

Commit:
Remove TS compatibility transition code (#85003)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/85003

Remove TS compatibility transition code

Test Plan: sandcastle

Reviewed By: cpuhrsch

Differential Revision: D39494677

fbshipit-source-id: c9e55376233b9acc4982fedd996e86bd8906c709

mikekgfb commented:
@pytorchbot merge
